### PR TITLE
Feature 1 cars

### PR DESCRIPTION
**Pull request by AndreBap**
Finished Feature-1-cars

### CODE REVIEW

File: basics/src/test/java/basics/CarTests.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import org.junit.Test;
+
+//@Test
+public class CarTests {
+	
+	@Test
+	public void canBuildCar() {
+		Car car = new Car("Corvette");
+		
+		Assert.assertNotNull(car);
```
Comment (dentaylor) [inline basics/src/test/java/basics/CarTests.java]: Put test sections into order by Arrange, Act, Assert

File: basics/src/test/java/basics/IEngine.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+public interface IEngine {
+	public int cylinderCount = 0;
+	
+	public void start();
+
```
Comment (dentaylor) [inline basics/src/test/java/basics/IEngine.java]: Remove whitespace

File: basics/src/test/java/basics/LargeEngine.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+public class LargeEngine implements IEngine {
+	public int cylinderCount = 8;
+	
+	public void start() {
+		System.out.println("Engine Started");
+	}
+
```
Comment (dentaylor) [inline basics/src/test/java/basics/LargeEngine.java]: Remove whitespace

File: basics/src/test/java/basics/Car.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+	}
+	
+	public int get_Odometer() {
+		return odometer;
+	}
+	
+	public void start() {
+		engine.start();
+	}
+
```
Comment (dentaylor) [inline basics/src/test/java/basics/Car.java]: RW (remove whitespace)

**Review by dentaylor — CHANGES_REQUESTED** [review]
Fix discrepancies then resubmit.

File: basics/src/test/java/basics/CarTests.java (hunk 1 above)
Comment (AndreBap) [reply basics/src/test/java/basics/CarTests.java]: Suggested changes were made. Should I close and make another pull request or continue with this?

Comment (dentaylor) [reply basics/src/test/java/basics/CarTests.java]: Keep this one

**Review by dentaylor — APPROVED** [review]

**Pull request merged**